src/commands/node.mjs

```diff
@@ -526,7 +526,33 @@ export class NodeCommand extends BaseCommand {
         title: 'Fetch platform software into network nodes',
         task:
           async (ctx, task) => {
-            return self.fetchPlatformSoftware(ctx, task, self.platformInstaller)
+            if (argv[flags.localBuildPath.name] !== '') {
```

Comment:
Please copy this new block of code into a local function like `uploadPlatformSoftware` and then setup the `subTasks` accordingly similar to how `fetchPlatformSoftware` is done.

Reply:
created new local function  `uploadPlatformSoftware`